### DISCUSSION TITLE
Example/Reference implementation for using JDBI3 in Dropwizard

### DISCUSSION THREAD

**Discussion by kavanpuranik** [dropwizard/dropwizard#8763] [Q&A] [answered]
We are currently looking into migrating from JDBI2 to JDBI3 in all our Dropwizard version 2.1.x services. An example implementation would be really helpful for migration.

I see an [example](https://github.com/dropwizard/dropwizard/tree/release/2.1.x/dropwizard-example) for Hibernate, but is there one for JDBI3?

**Comment by zUniQueX** [comment, marked as answer]
@kavanpuranik There is a dedicated [documentation page](https://www.dropwizard.io/en/stable/manual/jdbi3.html) related to JDBI3. Is there anything missing in the documentation you would like to have an example for? Then we should also add that to the docs.
